marshal.go

```diff
@@ -172,10 +173,16 @@ func (x *GoSNMP) sendOneRequest(packetOut *SnmpPacket,
 			break
 		}
 
+		dst, err := net.ResolveUDPAddr("udp", x.Target + ":" + strconv.Itoa(int(x.Port)))
+		if err != nil {
+			x.logPrintf("ERROR on ResolveUDPAddr: %s", err)
+			err = fmt.Errorf("Error on ResolveUDPAddr: %s", err.Error())
```

Comment:
Should we continue here to attempt another retry, or break to not rety?
```suggestion
			err = fmt.Errorf("Error on ResolveUDPAddr: %s", err.Error())
			continue;
```

Reply:
that's cool, code suggestions you can commit!

Reply:
woah